elements/lisk-transactions/src/utils/validation/validation.ts

```diff
@@ -164,9 +164,7 @@ export const isValidTransferData = (data: string): boolean =>
 	Buffer.byteLength(data, 'utf8') <= MAX_TRANSFER_ASSET_DATA_LENGTH;
 
 export const validateFee = (data: string) =>
-	isNumberString(data) &&
-	isGreaterThanZero(new BigNum(data)) &&
```

Comment:
Maybe check if it's `greater than or equal to zero` here?

Reply:
Added method `isGreaterThanOrEqualToZero` and its tests